build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         maven { url 'https://maven.google.com' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
Yeah everything is fine but why is this here

Reply:
Okay I undo the changes in few minutes